octobot_trading/enums.py

```diff
@@ -59,6 +59,20 @@ class OrderStatus(Enum):
     FILLED = "filled"
     CANCELED = "canceled"
     CLOSED = "closed"
+    UNKNOWN = "unknown"
+
+
+class OrderStates(Enum):
+    OPENING = "opening"
+    OPENED = "opened"
```

Comment:
I think we should use open (opened is the past form of the open verb, not the "not closed" meaning) https://dictionary.cambridge.org/grammar/british-grammar/open-or-opened

Reply:
:+1: